challenges/palindrome-detector.js

```diff
@@ -21,3 +21,17 @@
 */
 
 // YOUR CODE HERE
+
+function isPalindrome(str) {
+  var newStr = str.replace(/[^\w\s]/gi, '').replace(/ /g,'').toLowerCase(); //remove punctuation and spaces, convert to lowercas
+  let revString = ""
+  for (var i = newStr.length - 1; i >= 0; i-= 1 ) {
+    revString += newStr[i];
+  }
+
+  if (revString == newStr) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
While this works, you could replace lines 32-36 with just `return revString == newStr;`, since if it's true you return true, and otherwise you return false. :)